controllers/clusterconfig_controller.go

```diff
@@ -244,6 +244,25 @@ func (r *ClusterConfigReconciler) writeInputData(ctx context.Context, config *re
 			}
 		}
 
+		if config.Spec.ExtraManifestsRef != nil {
```

Comment:
Consider validating these files, at least the format.
Otherwise it might fail miserably later once we try to apply it on the cluster.